C++/C++.sublime-syntax

```diff
@@ -1303,6 +1328,9 @@ contexts:
     - match: '\bunion\b'
       scope: keyword.declaration.union.type.c++
       set: data-structures-union-definition
+    - match: '\bconcept\b'
+      scope: keyword.declaration.concept.c++
```

Comment:
To align with the other data structure definitions, `meta.concept` should also cover the `concept` keyword.

```suggestion
      scope: meta.concept.c++ keyword.declaration.concept.c++
```